Gemfile

```diff
@@ -21,6 +21,7 @@ gem "puma", "~> 3.4"
 gem "whenever", "~> 0.9"
 gem "validates_email_format_of", "~> 1.6"
 gem "redcarpet"
+gem "jbuilder", "~> 2.4"
```

Comment:
jbuilder is pretty awesome for rendering complex json structures. But yeah, if it's just a simple hash in the controller or view now, I think a `JSON.dump(our_ruby_object)` suffices at this moment :)